client/src/components/navbar/index.js

```diff
@@ -0,0 +1,64 @@
+import React ,{Component}from 'react';
+import ReactDOM from 'react-dom';
+import './style.css'    
+
+class Navbar extends Component{
+    constructor() {
+        super();
+        this.state={
+            activeTab: 'flights'
+        }
+
+        this.handleTabChange = this.handleTabChange.bind(this);
+    }
+
+    handleTabChange(activeTab) {
+        this.setState({
+            ...this.state,
+            activeTab
+        })
+    }
+
+    render(){
+        return(
+      <div>
+        <div className = "nav">
+            <div className ="user">
+            <a className="username" href={'#'} >username</a>
+                <img src={'/images/username.png'} />
+                
+            </div>
+            <div>
+            <img className ="plane" src={'/images/plan.png'} />
```

Comment:
If you're putting text inside `{...}` once, you should do it all the time.
Either write every `className` like `={"classname"}` or write all your `src` and `href` as `="text"`.